contributor-guides/writing-and-style-guide.md

```diff
@@ -102,6 +102,13 @@ This can be easier to follow (and perhaps more reassuring!) than a statement tha
 **After: add necessary context to the instruction, followed by opinionated usage**
 > Add the `LanguagePicker` component to your site in a component that is shown on every page. The example below adds this to the page footer:
 
+### Third-Party Instructions
+If a step in your guide involves a tool other than Astro (e.g. a CMS provider), keep the instructions functional and simple. Think of writing in a way that's still helpful even if the layout of the particular tool changes.  
+
+In that spirit, leave out details about another site's navigation and don't bother with screenshots.  
+
+If you want your reader to do something complex, consider linking to the relevant docs, like an official guide. 
```

Comment:
```suggestion
When documenting using Astro with third-party tools and services, focus on actions that are necessary **to integrate with Astro**. Do not document usage of products that we do not maintain (e.g. do not explain how to set up a new project in a CMS), but do feel free to include links to their own documentation where you believe it is helpful or necessary.

Do not include details about another site's navigation or settings. Your instructions should still be helpful even if the layout of the particular tool changes. However, do use the other site's vocabulary for features, options and settings so that your readers can search the site's own documentation for help if they need it.

If you want your reader to do something complex, such as create a new project in a CMS with pre-configured fields, consider making this entire action a Prerequisite.  You can link to the relevant docs, like an official guide, for completing this outside of Astro before your instructions begin. 
```
So, here's my initial take on this.  What do you think?

Nuances I added:
- emphasis on the fact that your steps should be about INTEGRATING WITH ASTRO whenever possible
- don't describe details but DO USE THEIR JARGON
- used a concrete example: "Creating a new project in a CMS"
- considering moving things to Prerequisites as a good place for linking to official docs/guides

Reply:
I like these ideas!

Reply:
No problem! I know how to give myself co-author credit if I  feel I deserve it.  ;)